web-client/src/providers/realUserMonitoring.ts

```diff
@@ -0,0 +1,30 @@
+import { AwsRum, AwsRumConfig } from 'aws-rum-web';
+
+export const initializeRealUserMonitoring = () => {
+  if (process.env.ENV === 'local') return;
+  let awsRum: AwsRum;
+  try {
+    const config: AwsRumConfig = {
+      allowCookies: true,
+      enableXRay: false,
+      endpoint: 'https://dataplane.rum.us-east-1.amazonaws.com',
+      identityPoolId: process.env.RUM_IDENTITY_POOL_ID,
+      sessionSampleRate: parseInt(process.env.RUM_SAMPLE_RATE!),
```

Comment:
I think `parseInt()` only likes whole integers!

```suggestion
      sessionSampleRate: Number(process.env.RUM_SAMPLE_RATE!),
```